code/__defines/misc.dm

```diff
@@ -315,4 +315,10 @@
 
 // Defines for fluorescence (/atom/var/fluorescent)
 #define FLUORESCENT_GLOWS   1	// Glows when under flourescent light
-#define FLUORESCENT_GLOWING 2	// Currently glowing due to flourescent light
\ No newline at end of file
+#define FLUORESCENT_GLOWING 2	// Currently glowing due to flourescent light
+
+// Flags used for utensil-food interaction.
+#define UTENSIL_FLAG_COLLECT  BITFLAG(0)
+#define UTENSIL_FLAG_SCOOP  BITFLAG(1)
+#define UTENSIL_FLAG_SLICE  BITFLAG(2)
```

Comment:
is `UTENSIL_FLAG_SLICE` really necessary when we use `has_edge()` for slicing foods? i'm worried that this is going to mean, for example, you can't slice foods with a survival knife or something like that